### DISCUSSION TITLE
希望添加输入文本进行翻译的方式

### DISCUSSION THREAD

**Discussion by 920922786** [immersive-translate/immersive-translate#1519] [Ideas]
### 新功能 | New Feature

1.希望添加输入文本进行翻译的方式，且可以选择与翻译网页不同的语种
2.希望添加为使用OpenAI等服务添加代理的方式访问，如设置一个代理地址及其Key后，可以用它进行访问。

### 为什么您想要添加这些功能 | Why you want to add these features

1.在访问外国网站时，有时需使用对应语言进行检索，添加此功能后，就无需特地打开在线翻译或安装翻译软件了。
2.有时在国内使用OpenAI会用购买代理服务的方式访问它，为了不再官方再付费买一次，因此希望增加此功能

### 提议 | Proposal

对于功能1，希望在浏览器扩展栏点击此扩展后的弹窗处增加一个选项卡的方式，单独设置其原文及译文，页面内设置上下两栏，上栏设置输入框，下栏设置译文内容，且译文可复制。

**Comment by theowenyoung** [comment]
我们有计划出一个独立的在线文本翻译计划，目前在我们的路线图中，应该会在 1 个月内上线：

![CleanShot 2024-03-12 at 04 02 38@2x](https://github.com/immersive-translate/immersive-translate/assets/62473795/e54d3da3-6b0e-4485-bdef-29be557dd69a)

> 2.希望添加为使用OpenAI等服务添加代理的方式访问，如设置一个代理地址及其Key后，可以用它进行访问。

目前已经支持，在 openai 的设置项中，点击【展开更多设置】即可。
